web/src/main/kotlin/com/example/ajaxproject/model/PrivateChatRoom.kt

```diff
@@ -1,12 +1,15 @@
 package com.example.ajaxproject.model
 
-import org.springframework.data.annotation.Id
+import com.fasterxml.jackson.annotation.JsonProperty
 import org.springframework.data.mongodb.core.mapping.Document
 
 @Document("private-chat-room")
 data class PrivateChatRoom(
-    @Id
+    @JsonProperty("id")
```

Comment:
Mongo model is about working with MONGO. Why do you need this annotation?

Reply:
Fixed